drivers/network/dd/ne2000/netne2000.inf

```diff
@@ -64,13 +64,11 @@ LoadOrderGroup = NDIS
 [Strings]
 ; Non-localizable
 ReactOS = "ReactOS Project"
-
-RealtekMfg = "Realtek Semiconductor Co., Ltd."
+RealtekMfg = "Realtek"
 
 ; Localizable
 GenericMfg = "(Generic network cards)"
 *PNP80D6.DeviceDesc = "NE2000-compatible Ethernet Adapter"
-
 *RTL8019.DeviceDesc = "Realtek RTL8019AS ISA Ethernet Adapter"
 PCI\VEN_10EC&DEV_8029.DeviceDesc = "Realtek RTL8029 PCI Ethernet Adapter"
```

Comment:
This additional line served a purpose to separate generic and specific manufacturer devices.

Reply:
Yes that was true in the past before you added the Localizable and the non-Localizable sections by your commit bbccad0ed60d619e
At this moment you created another aspect to split the strings. And since the newlines do not tell you which subsection withing the subsections they would end, this is getting very confusing. Therefore I decided to make your differentiation between Localizabe and Non-Localizable the only one. So the sepearator newlines do have a distinct purpose again. We should split only for a single aspect. Not multiple ones, as translators proved they cannot replicate that. It is really confusing if you separate with the same character different aspects within the subsections.

Reply:
This is your point, but I'm against doing this.

Reply:
![beforeAndAfter](https://user-images.githubusercontent.com/33393466/208683714-52a14e22-c2ee-49d0-b927-30a3386667b6.PNG)
Please watch this picture: Left side is before my change, and right side is after my change:
As you can see the left side is absolutely inconsistent in the splitting when compared to the translated sections below!

Reply:
The only inconsistency on left side of your screenshot is missing translations (for specific manufacturer devices). Otherwise it's consistent, as I said earlier, generic devices are separated from specific manufacturer devices. Only deletion on 113 line (left) is correct.

Reply:
Like I said: In many places the translators proved, that they are not able to replicate all multi-aspect-newline-separators. Some translators already stopped doing that entirely. For good reasons. It makes no sense to run into different directions. We should really use a consistent style.